config/cluster_config.yaml

```diff
@@ -1,9 +1,9 @@
 "product_family": "LC"
 "product_type": "k8s"
-#K8s based s3 server master details.
+# K8s based s3 server master details.
+# Note: Make sure master/worker credentials are same as we are using them to collect system stat.
 "nodes":
     - "hostname": ""
-      "ip": ""
```

Comment:
ip information is not needed? we can keep this param for future use as well

Reply:
We are not using it so removed for now. We can add back whenever needed.